src/otx/v2/adapters/torch/lightning/anomalib/modules/data/data.py

```diff
@@ -169,7 +169,12 @@ class OTXAnomalyDataModule(LightningDataModule):
         torch.Size([32, 3, 256, 256])
     """
 
-    def __init__(self, config: DictConfig | ListConfig, dataset: DatasetEntity, task_type: TaskType) -> None:
+    def __init__(
+            self,
+            config: DictConfig | ListConfig,
+            dataset: dict[Subset, DatumDataset],
+            task_type: TaskType,
+        ) -> None:
```

Comment:
again, if it is not intentional, it could be a single line?

Reply:
fixed at 0e4de99acf5e9b1e54d3375cba358c4c02d964b2.